setup.sh

```diff
@@ -15,8 +37,8 @@ git config --global core.editor "vim"
 #mkdir projects
 #cd projects
 #git clone git@github.com:TomPridham/tomfig.git
-mkdir $HOME/.config/autostart
-mkdir $HOME/.vim
+mkdir -p $HOME/.config/autostart
```

Comment:
`-p` is safer, if `~/.config` didn't exist yet for some reason `-p` would create it.